src/app/components/Item/ItemsContainer.jsx

```diff
@@ -90,9 +90,10 @@ class ItemsContainer extends React.Component {
       <ItemTable
         items={itemsToDisplay}
         bibId={bibId}
-        id="bib-item-table"
+        id='bib-item-table'
         searchKeywords={this.props.searchKeywords}
         holdings={this.props.holdings}
+        page='BibPage'
```

Comment:
Added `page='BibPage'` to this file. All other changes were performed by the formatter. Expect for the added `bib propType`, which was originally missing.